po/es.po

```diff
@@ -774,8 +774,7 @@ msgid ""
 "\"gnucash\">GnuCash</span> wikis."
 msgstr ""
 "Otros recursos son los wikis de <span class=\"gnucash\">GnuCash</span> en <a "
-"href=\"https://wiki.gnucash.org/wiki/\">inglés</a> o <a href=\"https://wiki."
-"gnucash.org/wiki/Es/\">español</a>."
+"href=\"https://wiki.gnucash.org/wiki/\">inglés</a>."
```

Comment:
@cyphra why do you not talk with me? wiki/Es was missing the redirect to Es/GnuCash. I have fixed it.

My suggestion: test it and revert this change.